frontend/package.json

```diff
@@ -4,28 +4,29 @@
   "private": true,
   "scripts": {
     "serve": "vue-cli-service serve",
-    "build": "vue-cli-service build",
+    "build": "npm run license-file-headers-check && vue-cli-service build",
     "test:unit": "vue-cli-service test:unit",
-    "lint": "vue-cli-service lint && npm run license-file-headers-add",
-    "prettify": "prettier --write **/*.{css,html,js,json,mjs,scss,ts,tsx,yml,md,vue}",
+    "lint": "vue-cli-service lint",
+    "prettify": "npm run license && prettier --write **/*.{css,html,js,json,mjs,scss,ts,tsx,yml,md,vue}",
```

Comment:
npm run license is also called in the build.yml will this cause issues?

Reply:
This most likely won't cause any issues as it only creates a new licenses/licenseInfos.json file. Since it won't be committed in the build, it does cause any problems. It actually is a good thing it's called in the build, since the result will always have the current dependency versions listed in the frontend.
The prettify script is currently for development and not part of a build chain (prettier calls should always happen before pushes and not on the pipeline).